src/App.vue

```diff
@@ -12,6 +12,15 @@
   </v-app>
 </template>
 
+<style>
+.v-main{
```

Comment:
Could you remove the "style" changes ?
- to avoid mixing column changes and style changes
- the color are already mostly in line with OFF. see #298